content/hi/docs/reference/glossary/container.md

```diff
@@ -0,0 +1,19 @@
+---
+title: कंटेनर (Container)
+id: container
+date: 2018-04-12
+full_link: /docs/concepts/containers/
+short_description: >
+   एक हल्की और पोर्टेबल एक्सेक्यूटबल योग्य इमेज जिसमें सॉफ़्टवेयर और उसकी सभी निर्भरताएँ शामिल हैं।
+
+aka: 
+tags:
+- fundamental
+- workload
+---
+ एक हल्की और पोर्टेबल एक्सेक्यूटबल योग्य इमेज जिसमें सॉफ़्टवेयर और उसकी सभी निर्भरताएँ शामिल हैं।
+
+<!--more--> 
+
+कंटेनर अलग-अलग क्लाउड या ओएस(OS) एनवायरनमेन्ट्स में और आसान स्केलिंग के लिए डिप्लॉयमेंट को आसान बनाने के लिए, अंतर्निहित होस्ट इन्फ्रास्ट्रक्चर से अनुप्रयोगों को अलग करते हैं।
+
```

Comment:
This PR is missing sentences present in `en` content. Please check : [https://github.com/kubernetes/website/blob/main/content/en/docs/reference/glossary/container.md](https://github.com/kubernetes/website/blob/main/content/en/docs/reference/glossary/container.md)